containers/runtime/driver/run.sh

```diff
@@ -19,8 +19,13 @@ if [ -n "${QPS_WORKERS_FILE}" ]; then
   export QPS_WORKERS=$(cat "${QPS_WORKERS_FILE}")
 fi
 
+
+if [ -f /var/data/qps_workers/server_target_override ]; then
```

Comment:
Why have the conditional if the script will fail anyway if the file is not present?

Reply:
I might have understand this wrong. -f is to check if the value of `$(cat /var/data/qps_workers/server_target_override)` can be passed to SERVER_TARGET_OVERRIDE. If I `cat` a non-existing file it would return an error and terminate the process. Regardless of having a value for SERVER_TARGET_OVERRIDE, the test would run. Inside the driver, there is a length check for the target provided through `--qps_server_target_override`, if the length is zero, the driver just ignore this flag. (https://github.com/grpc/grpc/blob/7c74d58cbecc8342c349cbf39eecb9aa0b666522/test/cpp/qps/driver.cc#L482) I might should be more specific to make it less confusing.